src/component/CategoryPicker.js

```diff
@@ -0,0 +1,125 @@
+import React, {Component} from 'react';
+import {View, StyleSheet, Text, Image, TouchableOpacity} from 'react-native';
+import * as Constants from '../common/constants';
+
+export default class CategoryPicker extends Component {
+    state = {open: false, selectedCategory: 'Tutte le categorie'};
+
+    renderChoices() {
+        return (
+            <View style={styles.choiceContainer}>
+                <TouchableOpacity
```

Comment:
Consider to use a single array for categories and iterate on it to create choices